src/stories/card.stories.js

```diff
@@ -35,18 +35,18 @@ export const Basic = () => html`
         of the card's content.
       </p>
       <cod-button
-        data-primary="true"
-        data-disable="false"
-        data-label="Link"
-        data-background-color="primary"
-        data-img=""
-        data-img-alt=""
-        data-icon=""
-        data-icon-order=""
-        data-icon-size=""
-        data-shape="fluid"
-        data-aria-label=""
-        data-link="https://detroitmi.gov"
+        primary="true"
+        disable="false"
```

Comment:
E.g. since these are boolean attributes, they should be like this now:

```html
<cod-button
  primary
  <!-- Notice how we don't even set `disabled` here since its omission should be treated like 'false'. -->
>
```